src/pymovements/events/ivt.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import annotations
+
+from typing import Optional
+import numpy as np
+
+from pymovements.transforms import vnorm
+from pymovements.events import Fixation
+
+
+def ivt(
+        x: list[list[float]] | np.ndarray,
+        v: Optional[list[list[float]] | np.ndarray],
+        t: float
+) -> list[Fixation]:
+    """
+    Identification of fixations based on velocity-threshold
+
+    Parameters
+    ----------
+    x: array-like
+        Continuous 2D position time series.
+    v: array-like
+        Corresponding continuous 2D velocity time series.
+    t: float
+        Velocity threshold.
```

Comment:
Maybe add description what exactly t is?

Reply:
velocity threshold in what unit? especially since the event(fixation) class has duration in samples

Reply:
do we need to specify which unit it actually is, as the unit is determined by the `velocity` argument?
so maybe just write that it's the same unit as for `velocity`